src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_private_link_scope.py

```diff
@@ -12,7 +12,8 @@ def __init__(self, method_name, config_file=None, recording_dir=None, recording_
         super(TestMonitorPrivateLinkScope, self).__init__(method_name)
         self.cmd('extension add -n application-insights')
 
-    @ResourceGroupPreparer(location='eastus2euap')
+    @record_only()  # record_only as the private-link-scope scoped-resource cannot find the components of application insights
+    @ResourceGroupPreparer(location='westus2')
```

Comment:
Why not change location back to 'eastus2euap'?

Reply:
`eastus2euap` is the wrong location for this service, we should correct it to let the true error expose.